pkgs/applications/virtualization/distrobox/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "distrobox";
-  version = "1.5.0.2";
+  version = "1.6.0";
```

Version 1.6.0.1 has been released:
```suggestion
  version = "1.6.0.1";
```

---

https://github.com/89luca89/distrobox/releases/tag/1.6.0.1